src/js/init.js

```diff
@@ -29,11 +29,6 @@ define(function (require) {
     var ui = require("./util/ui"),
```

Comment:
I find this renaming to be more confusing than it was before, but I guess it's all going away soon...